pkg/pillar/agentlog/agentlog.go

```diff
@@ -289,7 +289,7 @@ func RebootReason(reason string, bootReason types.BootReason, agentName string,
 	}
 
 	// Printing the reboot reason to the console
-	filename = "/dev/console"
```

Comment:
Some scripts from [https://github.com/lf-edge/eve/tree/master/pkg/pillar/scripts](pkg/pillar/scripts) also writes to /dev/console, doesn't make sense to change them as well?

Reply:
@rene I checked those and they do not affect monitor app. They actually should write to console so we see the output on /dev/ttyS* if connected. Monitor is started after they finish so it should be ok. I still need to modify getty to properly handle console devices anyways